CMakeLists.txt

```
@@ -59,6 +60,17 @@ if(USE_OPENMP)
   target_link_libraries(ensmallen INTERFACE OpenMP::OpenMP_CXX)
 endif()
 
+if(USE_BANDICOOT)
+  # Find Bandicoot and link it.
+  find_package(Bandicoot 0.100.0)
```

```suggestion
  find_package(Bandicoot 2.1.0)
```

I think this is the actual minimum version we require?